packages/label-sync/src/label-sync.ts

```diff
@@ -42,11 +42,25 @@ interface Repos {
 // from the local copy.  We are using the `PushEvent` to detect the change,
 // meaning the file running in cloud will be older than the one on master.
 let labelsCache: Labels;
-async function getLabels(github: GitHubAPI, repoPath: string) {
+async function getLabels(github: GitHubAPI, repoPath: string): Promise<Labels> {
   if (!labelsCache) {
     await refreshLabels(github, repoPath);
   }
-  return labelsCache;
+  const labels = {
+    labels: labelsCache.labels.slice(0),
+  } as Labels;
+  const apiLabelsRes = await handler.getApiLabels(repoPath);
```

Comment:
Little worried here. Isn't this essentially completely bypassing the cache?  Will this end up calling out to GCS constantly?

Reply:
It's only called once per function invocation, which only occurs rarely, e.g., when a new repo is created, or a label is deleted.

I think we're going to need to get rid of the `push` functionality any ways, because it takes like an hour to backfill all labels, and the maximum execution time of a cloud function is 9 minutes.